### ISSUE TITLE
Working in reproducible/replicable research environments with pyenv, pipenv and Jupyter notebooks

### ISSUE THREAD

**Issue by FlxVctr** [pyhamburg/talks#6]
#### Is this a talk you want to hold or are you interested in somebody talking about this topic?

I can give this talk.

#### In case you hold the talk, how long will the talk roughly be?

20 minutes

## Description

Not only as an academic, but also as a data scientist, it is important to make your results reproducable. At the same time, projects can lay dormant for years. Therefore, being able to effortlessly conserve/replicate the environment you've worked with back in the days is crucial. I would present the solution I found for my research during the last years.

**Comment by brakhane** [comment]
@FlxVctr Would you be able to hold this talk next week, 23rd?

**Comment by FlxVctr** [comment]
@brakhane Unfortunately not. Could do next month presumably.

**Comment by alexpovel** [comment]
Is this still active? Apart from COVID, of course.

[poetry](https://python-poetry.org/) could also be an interesting talking point.

**Comment by brakhane** [comment]
@alexpovel Yeah, we're still active, but no meetings due to COVID currently (we might do some online meetings soon). If you'd like to talk about poetry, create a new issue.

**Comment by FlxVctr** [comment]
I have looked at poetry, and actually started a project with it to try, but it rather seems to be aimed at actual software production (which is not necessarily bad but might overwhelm the average scientist/scholar). But I'd love to hear a talk about it!

**Comment by krenzlin** [comment]
+1 for poetry talk
Bit of topic, but I just discover PEP 517/518 and the pyproject.toml. Maybe this is already enough for some projects?
